lib/src/main/scala/nl.knaw.dans.easy.stage/Settings.scala

```diff
@@ -37,13 +37,15 @@ case class Settings(ownerId: String,
                     databaseUrl: String,
                     databaseUser: String,
                     databasePassword: String,
-                    licenses: Map[String, File]) {
+                    licenses: Map[String, File],
+                    includeBagMetadata: Boolean,
+                   ) {
   override def toString: String = {
     s"Stage-Dataset.Settings(ownerId = $ownerId, submissionTimestamp = $submissionTimestamp, " +
       s"bagitDir = $bagitDir, sdoSetDir = $sdoSetDir, urn = ${ urn.getOrElse("<not defined>") }, " +
       s"doi = ${ doi.getOrElse("<not defined>") }, otherAccessDoi = $otherAccessDoi, " +
       s"fileUris = $fileUris, state = $state, archive = $archive, " +
-      s"Database($databaseUrl, $databaseUser, ****))"
+      s"Database($databaseUrl, $databaseUser, ****), licenses = $licenses, includeBagMetadata = $includeBagMetadata)"
```

Comment:
Why no disciplines?

Reply:
How long do you want this `toString` to be? This `disciplines` map contains all 260-something discipline mappings from Fedora 😉

Reply:
How long do you want this `toString` to be? This `disciplines` map contains all 260-something discipline mappings from Fedora 😉